config/database_test.go

```diff
@@ -701,7 +701,7 @@ func TestDatabaseStoreLoad(t *testing.T) {
 		assert.Equal(t, map[string]interface{}{"SqlSettings": map[string]interface{}{"DataSourceReplicas": true}}, ds.GetEnvironmentOverrides())
 		// check that in DB config does not include overwritten variable
 		_, actualConfig := getActualDatabaseConfig(t)
-		assert.Equal(t, []string(nil), actualConfig.SqlSettings.DataSourceReplicas)
+		assert.Equal(t, []string{}, actualConfig.SqlSettings.DataSourceReplicas)
```

Comment:
@cpoile is this change correct? I needed it to get the tests passing, but honestly not sure if the change here means the test should actually be failing?

Reply:
Yep, that's good -- the test should pass (the original DB config should have an empty slice in `DataSourceReplicas`).